pkg/util/tls_config.go

```diff
@@ -86,25 +88,25 @@ func GetMinTLSVersion() uint16 {
 	return minTLSVersion
 }
 
-func initMinTLSVersion(logger logr.Logger) uint16 {
-	version, found := os.LookupEnv("KEDA_HTTP_MIN_TLS_VERSION")
-	minVersion := tls.VersionTLS12
-	if found {
-		switch version {
-		case "TLS13":
-			minVersion = tls.VersionTLS13
-		case "TLS12":
-			minVersion = tls.VersionTLS12
-		case "TLS11":
-			minVersion = tls.VersionTLS11
-		case "TLS10":
-			minVersion = tls.VersionTLS10
-		default:
-			logger.Info(fmt.Sprintf("%s is not a valid value, using `TLS12`. Allowed values are: `TLS13`,`TLS12`,`TLS11`,`TLS10`", version))
-			minVersion = tls.VersionTLS12
-		}
+func initMinTLSVersion() (uint16, error) {
+	version, _ := os.LookupEnv("KEDA_HTTP_MIN_TLS_VERSION")
+
+	switch version {
+	case "":
+		minTLSVersion = tls.VersionTLS12
+	case "1.0", "TLS10":
+		minTLSVersion = tls.VersionTLS10
+	case "1.1", "TLS11":
+		minTLSVersion = tls.VersionTLS11
+	case "1.2", "TLS12":
+		minTLSVersion = tls.VersionTLS12
+	case "1.3", "TLS13":
```

Comment:
Doesn using `1.X` make sense?

Reply:
It has been inherited when we decided to couple the TLS version parsing together and then decided to drop the CLI flag support.